test/unit/voldemort/client/OfflineStateTest.java

```diff
@@ -262,4 +268,23 @@ public void testStateTransitions() throws InterruptedException {
         assertTrue(testOnlineTraffic());
         assertTrue(testSlopStreaming());
     }
+
+    @Test
```

Comment:
I would also like to add one more test, where a server stopped in online mode, can be brought up in offline mode. The steps that are done in the test, need to be documented and given to SREs.